app/Actions/WebAuth/VerifyAuthentication.php

```diff
@@ -46,7 +46,8 @@ protected function getUserFromCredentials(array $credentials): ?User
 		// authenticator may use to sign the subsequent challenge by the server.
 		if ($this->isSignedChallenge($credentials)) {
 			$id = $this->binaryID($credentials['rawId']);
-			if ($id) {
+			if ($id != '') {
```

Comment:
```suggestion
			if ($id !== '') {
```

Why the weak comparison?